mocks/mocks.py

```diff
@@ -18,15 +18,131 @@
 import gunagala
 from gunagala.utils import ensure_unit
 
-from mocks.utils import load_yaml_config
+from mocks import utils
```

Comment:
I think the old way is preferred: `from mocks.utils import load_yaml_config` since it only imports the function that is needed.